### ISSUE TITLE
Millilitres - ml or mL

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual-community-backlog#506]

**Comment by albfreeman** [comment]
In NHS.UK content, we've always abbreviated millilitres in lowercase as ml. We’ve been using it for several years, particularly in our medicines content.

But we had some feedback from a pharmacist saying that the standard abbreviation is mL for safety reasons.

We discussed it in the content team and felt that lowercase is what the public are more used to, so we should stick to that. People will regularly see millilitres abbreviated in lowercase on medicine packaging and in shops.

It may be that mL is more commonly used by doctors or pharmacists, for example in a patient’s medical notes or on prescriptions. 

Unless we get more evidence to the contrary, we feel ml is better in our public-facing content.

The UK Metric Association says in their style guide that either ml or mL is valid. https://ukma.org.uk/style-guide/

**Comment by KarinMochan** [comment]
I agree - this is what we were taught in school and what we're used to seeing elsewhere, either online or in print. 
The abbreviation mL, while correct in prescriptions etc, might look like a typo in public-facing content.
